ui-v2/tests/acceptance/dc/acls/policies/navigate.feature

```diff
@@ -0,0 +1,15 @@
+@setupApplicationTest

```

Comment:
Didn't we call this file `navigation.feature` in the other PRs? Probably best to keep everything consistent.

Reply:
Yes 😅 . I'll update the name.